tests/unit/test_xblock.py

```diff
@@ -3,22 +3,24 @@
 
 import ddt
 import mock
+
 from xblock.core import XBlock
 from xblock.field_data import DictFieldData
 from xblock.fragment import Fragment
 
+from dalite_xblock import dalite_xblock
 from dalite_xblock.dalite_xblock import DaliteXBlock
-from dalite_xblock.utils import DaliteLtiPassport
+from dalite_xblock.passport_utils import DaliteLtiPassport
 from tests.utils import TestWithPatchesMixin
 
 DEFAULT_LTI_PASSPORTS = [
-    "(dalite-xblock)dalite-ng-1;http://first.url:8080/;KEY;SECRET",
-    "(dalite-xblock)dalite-ng-2;https://other.url:8080;OTHERKEY;OTHERSECRET"
+    "dalite-ng-1:dalite-xblock:aHR0cDovL2ZpcnN0LnVybDo4MDgwO0tFWTtTRUNSRVQ=",
+    "dalite-ng-2:dalite-xblock:aHR0cDovL290aGVyLnVybDtPVEhFUktFWTtPVEhFUlNFQ1JFVA=="
 ]
 
 PARSED_LTI_PASSPORTS = {
-    "dalite-ng-1": DaliteLtiPassport("dalite-ng-1", "http://first.url:8080/", "KEY", "SECRET"),
-    "dalite-ng-2": DaliteLtiPassport("dalite-ng-2", "https://other.url:8080", "OTHERKEY", "OTHERSECRET"),
+    "dalite-ng-1": DaliteLtiPassport("dalite-ng-1", "http://first.url:8080", "KEY", "SECRET"),
+    "dalite-ng-2": DaliteLtiPassport("dalite-ng-2", "http://other.url", "OTHERKEY", "OTHERSECRET"),
```

Comment:
Added this case and a bunch of other here: https://github.com/open-craft/xblock-dalite/pull/2/commits/2f0039a07a0efca9baf21dafa4af06b9cd554e76#diff-6ac57e7e8afa8bd1d6836a7f77908adeR20